migrate/v0_11/migrate.go

```diff
@@ -65,3 +78,148 @@ func MigrateBep3(oldGenState v0_9bep3.GenesisState) v0_11bep3.GenesisState {
+				DelegatedFree:    acc.BaseVestingAccount.DelegatedFree,
+				DelegatedVesting: acc.BaseVestingAccount.DelegatedVesting,
+				EndTime:          acc.BaseVestingAccount.EndTime,
+			}
+			var newPeriods v39_1vesting.Periods
+			for _, p := range acc.VestingPeriods {
+				newPeriods = append(newPeriods, v39_1vesting.Period(p))
+			}
+			pva := v39_1vesting.PeriodicVestingAccount{
+				BaseVestingAccount: &bva,
+				StartTime:          acc.StartTime,
+				VestingPeriods:     newPeriods,
+			}
+			newAccounts = append(newAccounts, v39_1authexported.GenesisAccount(&pva))
+
+		case *v38_5supply.ModuleAccount:
+			ba := v39_1auth.BaseAccount(*(acc.BaseAccount))
+			ma := v39_1supply.ModuleAccount{
+				BaseAccount: &ba,
+				Name:        acc.Name,
+				Permissions: acc.Permissions,
+			}
+			newAccounts = append(newAccounts, v39_1authexported.GenesisAccount(&ma))
+
+		case *v0_9validator_vesting.ValidatorVestingAccount:
+			ba := v39_1auth.BaseAccount(*(acc.BaseAccount))
+			bva := v39_1vesting.BaseVestingAccount{
+				BaseAccount:      &ba,
+				OriginalVesting:  acc.BaseVestingAccount.OriginalVesting,
+				DelegatedFree:    acc.BaseVestingAccount.DelegatedFree,
+				DelegatedVesting: acc.BaseVestingAccount.DelegatedVesting,
+				EndTime:          acc.BaseVestingAccount.EndTime,
+			}
+			var newPeriods v39_1vesting.Periods
+			for _, p := range acc.VestingPeriods {
+				newPeriods = append(newPeriods, v39_1vesting.Period(p))
+			}
+			pva := v39_1vesting.PeriodicVestingAccount{
+				BaseVestingAccount: &bva,
+				StartTime:          acc.StartTime,
+				VestingPeriods:     newPeriods,
+			}
+			var newVestingProgress []v0_11validator_vesting.VestingProgress
+			for _, p := range acc.VestingPeriodProgress {
+				newVestingProgress = append(newVestingProgress, v0_11validator_vesting.VestingProgress(p))
+			}
+			vva := v0_11validator_vesting.ValidatorVestingAccount{
+				PeriodicVestingAccount: &pva,
+				ValidatorAddress:       acc.ValidatorAddress,
+				ReturnAddress:          acc.ReturnAddress,
+				SigningThreshold:       acc.SigningThreshold,
+				CurrentPeriodProgress:  v0_11validator_vesting.CurrentPeriodProgress(acc.CurrentPeriodProgress),
+				VestingPeriodProgress:  newVestingProgress,
+				DebtAfterFailedVesting: acc.DebtAfterFailedVesting,
+			}
+			newAccounts = append(newAccounts, v39_1authexported.GenesisAccount(&vva))
+
+		default:
+			panic(fmt.Sprintf("unrecognized account type: %T", acc))
+		}
+	}
+
+	// ---- add harvest module accounts -------
+	lpMacc := v39_1supply.NewEmptyModuleAccount(v0_11harvest.LPAccount, v39_1supply.Minter, v39_1supply.Burner)
+	lpMacc.SetCoins(sdk.NewCoins(sdk.NewCoin("hard", sdk.NewInt(80000000000000))))
+	newAccounts = append(newAccounts, v39_1authexported.GenesisAccount(lpMacc))
+	delegatorMacc := v39_1supply.NewEmptyModuleAccount(v0_11harvest.DelegatorAccount, v39_1supply.Minter, v39_1supply.Burner)
+	delegatorMacc.SetCoins(sdk.NewCoins(sdk.NewCoin("hard", sdk.NewInt(40000000000000))))
```

Comment:
nit: linters complaining about unchecked return values on `SetCoins`, could check and panic